problems/[23.08.21]2766. Relocate Marbles.js

```diff
@@ -6,14 +6,14 @@
  */
 //시간복잡도 O(n*m) n = nums의 길이, m = moveFrom, moveTo의 길이
 const relocateMarbles = function (nums, moveFrom, moveTo) {
-  let result = [...nums];
+  const result = [...nums];
 
   for (let i = 0; i < moveFrom.length; i++) {
-    if (result.indexOf(moveFrom[i]) !== -1) {
-      let resultIndex = result.indexOf(moveFrom[i]);
+    while (result.indexOf(moveFrom[i]) !== -1) {
```

Comment:
`while` 반복문의 경우 어떠한 경우에도 반복이 되지 않는 것 같습니다.
이런 경우에는 처음에 작성하신 `if`가 더 적절해 보이네요.
```suggestion
    if (result.indexOf(moveFrom[i]) !== -1) {
```

Reply:
만약 같은 숫자가 있는경우 두개 모두 바꾸어 주어야 하는 케이스가 있어서 while 문을 사용했습니다.